variables.tf

```diff
@@ -115,3 +115,39 @@ variable "timeouts" {
     delete = "10m"
   }
 }
+
+variable "autoscaling_defaults" {
+  description = "A map of default autoscaling settings"
+
+  type = object(
```

Comment:
The last thing is. Could you change the type to `map(string)`? Objects are too strict (I don't think we have to use them in any terraform-aws-modules).

Reply:
That's the point of using an object, you can specify types of the keys. But no worries, I changed it.